src/views/Main/Catalog/Components/PropertyTypeList.js

```diff
@@ -82,13 +86,11 @@ export class PropertyTypeList extends React.Component {
     });
   }
 
-  addProperty = () => (this.props.navBar ? this.createNewPropertyType() : this.addPropertyToSchema());
-
   createNewPropertyType = () => {
     const id = this.props.id;
     const name = document.getElementById('pName'.concat(id)).value;
     const namespace = document.getElementById('pNamespace'.concat(id)).value;
-    const datatype = document.getElementById('pDatatype'.concat(id)).value;
+    const datatype = document.getElementById('pDatatype'.concat(id)).firstChild.firstChild.value;
```

Comment:
can't you get the value from the component instead of having to go through the DOM?